modules/mobile/app/helpers/mobile/pagination_helper.rb

```diff
@@ -0,0 +1,71 @@
+module Mobile
```

Comment:
Might be worth adding explicit tests for this rather than just relying on request specs.